hazelcast/src/test/java/com/hazelcast/test/HazelcastParallelClassRunner.java

```diff
@@ -59,6 +60,8 @@ private static int getDefaultMaxThreads() {
             // on parallelism inside each JVM. otherwise it's easy to use too much resource
             // and the test duration is actually longer and not shorter.
             cpuWorkers = min(4, cpuWorkers);
+            // decrease the amount of resources used when running in multiple JVM
+            RuntimeAvailableProcessors.overrideDefault(8);
```

Comment:
I don't like *change-something-underhood*  logic in `get*` method. This could be resolved by using a simple `static` block instead of the static `get` method. The method here is used just to configure a constant.
The second concern is about using the hardcoded value 8. What if the real number of available processors is lower? Shouldn't we use the `min()` function here?